src/Infrastructure/Repositories/DepartmentRepository.cs

```diff
@@ -0,0 +1,23 @@
+using System.Data;
+using Application.Common.Interfaces;
+using Dapper;
+using Domain.Entities;
+
+namespace Infrastructure.Repositories;
+
+public class DepartmentRepository : IDepartmentRepository
+{
+    private readonly IDbConnection _connection;
+    private readonly IDbTransaction _transaction;
+
+    public DepartmentRepository(IDbConnection connection, IDbTransaction transaction)
```

Comment:
what's the point of injecting without assigning it? and is it needed here now?

Reply:
I resolve it in the latest commit

Reply:
ok